### DISCUSSION TITLE
FastDoom 1.0.3

### DISCUSSION THREAD

**Discussion by viti95** [viti95/FastDoom#215] [General]
* Optimized rendering of sky and player sprite in 320x200 and 320x240 modes
* Added support for id Anthology versions of TNT and Plutonia
* Removed -bfg command line (not supported officially)
* Updated development environment script so it doesn't fail on VoidLinux

<hr /><em>This discussion was created from the release <a href='https://github.com/viti95/FastDoom/releases/tag/1.0.3'>FastDoom 1.0.3</a>.</em>

**Comment by RedIce25** [comment]
Made issue #216, seems to crash on the lift in Doom2 MAP06.
